zebrad/tests/acceptance.rs

```diff
@@ -500,6 +500,22 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[ignore]
+fn sync_one_checkpoint() -> Result<()> {
+    zebra_test::init();
+
+    let mut child = testdir()?
+        .with_config(persistent_test_config()?)?
+        .spawn_child(&["start"])?
+        .with_timeout(Duration::from_secs(20));
```

Comment:
I wonder if this timeout is too short - I guess we'll find out after we run the tests some more.

Reply:
yolo!

Reply:
It seems to work fine on the testnet, see PR #1035.

I also saw a crash due to out of order blocks, but that will be fixed by #1011.